src/ffi/pyframe.rs

```diff
@@ -8,5 +8,4 @@ opaque_struct!(PyFrameObject);
 extern "C" {
     pub fn PyFrame_GetLineNumber(f: *mut PyFrameObject) -> c_int;
 }
-// skipped PyFrame_GetLineNumber
```

Comment:
This symbol is still present in the C header? 

https://github.com/python/cpython/blob/a40675c659cd8c0699f85ee9ac31660f93f8c2f5/Include/pyframe.h#L15

Reply:
Maybe I get it wrong, but isn't there a `PyFrame_GetLineNumber` at line 9?

Reply:
Haha yes sorry that's my bad for reviewing at midnight 😮‍💨